modules/api/src/main/java/org/apache/ignite/lang/NullableValue.java

```diff
@@ -26,11 +27,50 @@
  * @param <T> Value type.
  * @see org.apache.ignite.table.KeyValueView#getNullable(Transaction, Object)
```

Comment:
I don't think that it is a good idea to refer to a `tx` package from `lang` package. It is the calls of general purpose and it could be used al alternative to `Optional` with relaxed requirement for return value. Better documentation should note it.